doc/manual/src/release-notes/rl-next.md

```diff
@@ -21,3 +21,17 @@
 * Error traces have been reworked to provide detailed explanations and more
   accurate error locations. A short excerpt of the trace is now shown by
   default when an error occurs.
+
+* On Linux, `nix develop` now sets the
+  [*personality*](https://man7.org/linux/man-pages/man2/personality.2.html)
+  for the development shell in the same way as the actual build of the
+  derivation. This makes shells for `i686-linux` derivations work
+  correctly on `x86_64-linux`.
+
+* New command: `nix store check-trusted-user`, which checks if you are a
+  trusted-user, returning 1 or 0. This is useful for scripting interactions with
+  remote Nix stores.
+
+* `nix doctor` now prints `[INFO]` instead of just `[PASS]` or `[FAIL]` and will
+  use the `nix store check-trusted-user` function to print whether you are a
+  trusted-user or not.
```

Comment:
I find this point a bit hard to read (and technically misleading since `nix doctor` doesn't really call `nix store check-trusted-user`). What about simply:

```suggestion

* `nix doctor` now displays whether you are a trusted user or not.
```

Reply:
I like that more. Fixed.